src/cow-react/common/utils/fetch.ts

```diff
@@ -0,0 +1,28 @@
+import { backOff, BackoffOptions } from 'exponential-backoff'
+
+// See config in https://www.npmjs.com/package/@insertish/exponential-backoff
+const DEFAULT_BACKOFF_OPTIONS: BackoffOptions = {
+  numOfAttempts: 10,
+  maxDelay: Infinity,
```

Comment:
I think we should set a max amount here or requests might be hanging forever
What about 5min?

Reply:
it doesn't, but i will document better. The maximum is a bit more than 100s, this is because its only doing 10 attempts by default